lldb/source/Target/SwiftLanguageRuntimeDynamicTypeResolution.cpp

```diff
@@ -440,17 +433,16 @@ class LLDBMemoryReader : public swift::remote::MemoryReader {
                   std::string &dest) override {
     Log *log(GetLogIfAllCategoriesSet(LIBLLDB_LOG_TYPES));
 
-    if (log)
-      log->Printf(
-          "[MemoryReader] asked to read string data at address 0x%" PRIx64,
-          address.getAddressData());
+    LLDB_LOGV(log,
+              "[MemoryReader] asked to read string data at address 0x%" PRIx64,
+              address.getAddressData());
 
     Target &target(m_process.GetTarget());
     Address addr(address.getAddressData());
     Status error;
     target.ReadCStringFromMemory(addr, dest, error);
     if (error.Success()) {
-      if (log) {
+      auto format_string = [](const std::string &dest) {
```

Comment:
I used a lambda here to make its execution lazy, it'll only run when logging verbosely.